sdcm/utils/common.py

```diff
@@ -2966,3 +2966,11 @@ def __exit__(self, exit_type, value, _traceback):
 def time_period_str_to_seconds(time_str: str) -> int:
     """Transforms duration string into seconds int. e.g. 1h -> 3600, 1h22m->4920 or 10m->600"""
     return sum([int(g[0] or 0) * 3600 + int(g[1] or 0) * 60 + int(g[2] or 0) for g in duration_pattern.findall(time_str)])
+
+
+def sleep_for_percent_of_duration(duration: int, percent: int, min_duration: int, max_duration: int):
+    """Waits the percentage of a duration in seconds, with a minimum and maximum duration"""
```

Comment:
Worth to explain it like min < duration * percentage < max, it's hard to understand it like this.

Reply:
done